### DISCUSSION TITLE
Confirming SuperAdmin lifetime <= CA lifetime

### DISCUSSION THREAD

**Discussion by Credentive-Sec** [Keyfactor/ejbca-ce#210] [General]
Because I'm running this in a lab, I set the EJBCA lifetime to be quite short (1 year). I'm assuming that the SuperAdmin lifetime must be less than or equal to the CA lifetime. Is that a safe assumption?

**Comment by mike-agrenius-kushner** [comment]
Indeed, the Management CA (or any CA) cannot issue a certificate longer than its own expiry date.

Cheers,
Mike
